src/libutil/types.hh

```diff
@@ -14,6 +14,7 @@ using std::list;
 using std::set;
 using std::vector;
 using std::string;
+using std::string_view;
```

Comment:
Let's not add more of these. In fact I'd rather get rid of the aliases for {set, vector, string}.

Reply:
we'll remove it. should `PathView` also go away to match, or would that be an exception?

Reply:
`PathView` is okay though maybe we should get rid of `Path` eventually, since it doesn't really serve a purpose.